project/plugins.sbt

```diff
@@ -1 +1,2 @@
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
+addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")
```

Comment:
Do we want this?

Reply:
[The fine manual says we do](https://github.com/sbt/sbt-pgp#setup)!  I imagine the reasoning is something like this...
> _I know I can sign and publish with sbt-pgp version 2.1.2.  I might not be able to sign and publish with some other version of sbt-pgp.  So let's drag it in._
Given that we do the same with sbt-assembly, which also says to include it, I think this makes sense.